pom.xml

```diff
@@ -80,12 +81,14 @@
 				<groupId>org.sonarsource.analyzer-commons</groupId>
 				<artifactId>sonar-analyzer-commons</artifactId>
 				<version>${sonar-analyzer-commons.version}</version>
+				<scope>compile</scope>
```

Comment:
please, delete this scope. no need to specify it. I tested locally without it and it's ok.

Reply:
done